bitnami/dataplatform-bp1/values-metrics.yaml

```diff
@@ -126,6 +126,13 @@ kafka:
         requests: 
           cpu: 100m
           memory: 128Mi
+      ## Service configuration
```

Comment:
Please add to this file only the parameters that need to be changed instead of copying the full `values.yaml`